test/README.md

```diff
@@ -66,3 +66,8 @@ mode, use
 ```bash
 test/local-e2e-tests.sh -short
 ```
+
+## Test images
+
+Please refer this [link](https://github.com/knative/serving/blob/master/test/README.md#test-images) on how 
+to add/upload test images
```

Comment:
its fine to copy over the docs from serving to here

Reply:
copied the test images part alone. Is that ok ?